rct229/rulesets/ashrae9012019/section19/section19rule18.py

```diff
@@ -0,0 +1,366 @@
+                )
+            else:
+                undetermined_msg = (
+                    f"Fan power for {hvac_id_b} is greater than expected per Section and Table G3.1.2.9 assuming no pressure drop adjustments (e.g., sound attenuation, air filtration, fully ducted return "
+                    f"when required by code, airflow control devices, carbon and other gas-phase air cleaners, coil runaround loops, evaporative humidifier/coolers in series with another cooling coil, "
+                    f"exhaust systems serving fume hoods, and laboratory and vivarium exhaust systems in high-rise buildings) per Table 6.5.3.1-2 other than {black_word} and "
+                    f"MERV filters defined in the RMD (if modeled). Expected Wattage = {expected_fan_wattage_b.to(ureg.kW)} kW. however not all pressure drop adjustments are able to be captured in the RMD so conduct manual check."
+                )
+
+            return undetermined_msg
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            more_than_one_exhaust_fan_and_energy_rec_is_relevant_b = calc_vals[
+                "more_than_one_exhaust_fan_and_energy_rec_is_relevant_b"
+            ]
+            more_than_one_supply_fan_b = calc_vals["more_than_one_supply_fan_b"]
+            total_fan_power_b = calc_vals["total_fan_power_b"]
+            expected_fan_wattage_b = calc_vals["expected_fan_wattage_b"]
+
+            return not more_than_one_exhaust_fan_and_energy_rec_is_relevant_b and (
+                (
+                    not more_than_one_supply_fan_b
+                    and std_equal(total_fan_power_b, expected_fan_wattage_b)
+                )
+                or (total_fan_power_b < expected_fan_wattage_b)
+            )
+
+        def get_pass_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.BASELINE_0
+            hvac_id_b = hvac_b["id"]
+            more_than_one_exhaust_fan_and_energy_rec_is_relevant_b = calc_vals[
+                "more_than_one_exhaust_fan_and_energy_rec_is_relevant_b"
+            ]
+            total_fan_power_b = calc_vals["total_fan_power_b"]
+            expected_fan_wattage_b = calc_vals["expected_fan_wattage_b"]
+            min_fan_wattage_b = calc_vals["min_fan_wattage_b"]
+
+            pass_msg = ""
+            if (
+                not more_than_one_exhaust_fan_and_energy_rec_is_relevant_b
+                and total_fan_power_b < expected_fan_wattage_b
+            ):
+                pass_msg = (
+                    f"The total fan power for {hvac_id_b} is modeled as {total_fan_power_b.to(ureg.kW)} kW which is less than the expected including pressure drop adjustments "
+                    f"for exhaust air energy recovery and MERV filters as applicable which was calculated as {min_fan_wattage_b.to(ureg.kW)} kW. Pass because this is generally considered more conservative."
+                )
+
+            return pass_msg
```

Comment:
We do not need this case for PASS message.

Reply:
Addressed